components/Days.vue

```diff
@@ -67,11 +67,13 @@ export default {
   display: none;
 }
 .days {
-  background-color: #f8f8f8;
+  background-color: var(--background-card);
   display: flex;
   justify-content: space-around;
   align-items: center;
   padding: 35px 60px;
+  border: var(--border-color) solid;
```

Comment:
Could we combine together, please :)
`border: 1px var(--border-color) solid;`
https://developer.mozilla.org/en-US/docs/Web/CSS/border